app/views/devise/shared/_form_wrap.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="h-full bg-gradient-to-tl from-green-400 to-indigo-900 w-full py-16 px-4">
+  <div class="w-full max-w-md mx-auto my-6">
+    <div class="bg-white shadow p-3 mb-5 bg-whit rounded px-8 pt-6 pb-8">
+    
+      <%= yield :form_wrap %>
+    </div>
+  </div>
+</div>
```

Comment:
please add a blank line in the end of this file.

Reply:
Thanks Juan!One question,why do we need a blank line at the end?

Reply:
In systems like Linux, a blank space is left, since lines of code can be added from the command line and if there is no blank line, some code can be left behind that causes errors:

Example:

```
endif
```

Reply:
Thank you Yil!